app/controllers/index.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+
+get '/' do
+  'Sealions'
+end
+
+#Create
+post '/sealions' do
+  Sealion.create(:name => params[:name])
```

Comment:
@laurennicoleroth Great work! What if we wanted to redirect to the SHOW  route of the sealion we've just created? How would the redirect uri change?